src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java

```diff
@@ -263,44 +259,54 @@ public String getCorrespondenceAddress() {
                 )
                 .filter(value -> value != null && !value.isEmpty())
                 .collect(joining("\n"));
+        } else {
+            return Stream.of(
+                    address.getAddressLine1(),
+                    address.getAddressLine2(),
+                    address.getAddressLine3(),
+                    address.getPostTown(),
+                    address.getCounty(),
+                    address.getCountry(),
+                    address.getPostCode()
+                )
+                .filter(value -> value != null && !value.isEmpty())
+                .collect(joining("\n"));
         }
+    }
 
-        return null;
+    @JsonIgnore
+    public YesOrNo getCorrespondenceAddressOverseas() {
```

Comment:
could this have an Is in it so it's obvious it is a flag? Like CorrespondenceAddressIsOverseas

Reply:
Yep.  Done.